### PR TITLE
Contact section Added

### PR DESCRIPTION
**Pull request by EddxSotz**
For this stage of the project I: 
-Added contact form
-Collected form data by using the Formspree service
-Made use of Grid and Flexbox
-Modified CSS styles

### CODE REVIEW

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @EddxSotz , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that hasn't any commit from your partner kindly contact your partner and ask for commit because this milestone is a pair programming activity.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/styles.css [hunk 1]
```diff
@@ -1,413 +1,538 @@
...
+  margin-inline: auto;
+  font-size: var(--button_font_size);
+  color: var(--button_font_color);
+}
+
+#contact_button:hover {
+  background-color: white;
+  border: solid 2px #ff8843;
+  cursor: pointer;
+}
```
Comment (divinecharlotte) [inline css/styles.css]: - [ ] kindly consider giving this button color so as the text it contains will keep being visible when a person hovers it
- your design 👇 

- Figma design 👇

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
# Hello @EddxSotz      👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- GitHub Flow observed ✔️
- Good commit messages ✔️
- Descriptive Readme 👍 
## Required Changes ♻️
consider the comments under the codes

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @EddxSotz   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**